charts/terraform-cloud-operator/values.yaml

```diff
@@ -22,6 +22,9 @@ operator:
   # List of namespaces the controllers should monitor
   watchedNamespaces: []
 
+  # The API URL of a TFE instance
+  tfe_address: ""
```

Comment:
Should we use here camelCase notation to have consistency in names?

Reply:
Definitely camel case, this was me copy-pasting blindly.